apps/dashboard/app/helpers/active_jobs_helper.rb

```diff
@@ -58,4 +58,36 @@ def has_ganglia(host)
   def has_grafana(host)
     OODClusters[host].try { |cluster| cluster.custom_allow?(:grafana) } || false
   end
+
+  def status_label(status)
+    case status
+    when "completed"
+      label = "Completed"
+      labelclass = "badge-success"
+    when "running"
+      label = "Running"
+      labelclass = "badge-primary"
+    when "queued"
+      label = "Queued"
+      labelclass = "badge-info"
+    when "queued_held"
+      label = "Hold"
+      labelclass = "badge-warning"
+    when "suspended"
+      label = "Suspend"
+      labelclass = "badge-warning"
+    else
+      label = "Undetermined"
+      labelclass = "badge-default"
+    end
+    "<div style='white-space: nowrap;'><span class='badge #{labelclass}'>#{label}</span></div>".html_safe
+  end
+
+  def filters
+    ::ActiveJobs::Filter.list
+  end
+
+  def default_filter_id
+    ::ActiveJobs::Filter.default_id.to_s
+  end
```

Comment:
These methods are completely new. The views now use these methods instead of using `Filter.list` or `Filter.default_id.to_s` directly.